generators/server/templates/src/main/java/package/config/_UaaConfiguration.java

```diff
@@ -87,9 +91,8 @@ public void configure(ClientDetailsServiceConfigurer clients) throws Exception {
          */
         clients.inMemory()
             .withClient("web_app")
-            .scopes("web-app")
+            .scopes("openid")
```

Comment:
It is used, when you generate an gateway app using uaa option. Just take a look at the token request. But you can keep it this way. I can fix this also, if you want